packages/builder/src/pages/builder/app/[application]/updating.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import Spinner from "components/common/Spinner.svelte"
+  import { API } from "api"
+
+  setInterval(async () => {
+    const response = await API.get({ url: "/api/migrations/status" })
+    if (!response.migrated) {
+      return
+    }
+
+    const urlParams = new URLSearchParams(window.location.search)
+    const returnUrl = urlParams.get("returnUrl")
```

Comment:
Routify provides URL and query params via the `$params` store, so we should make use of those. I'd imaging that simply `$params.returnUrl` here is the value we're after.
https://www.routify.dev/guide/introduction/parameters

Reply:
Added the usage. For some reason it's not stripping the `?` properly. Any idea why?